components/builder-api/src/server/resources/projects.rs

```diff
@@ -220,6 +240,7 @@ fn create_project(req: HttpRequest,
 #[allow(clippy::needless_pass_by_value)]
 fn get_project(req: HttpRequest,
                path: Path<(String, String)>,
+               body: Option<Json<ProjectGetReq>>,
```

Comment:
So that is actually how I had written this before. I refactored to bring get and delete in line with the previously defined behavior of create and update. Should we convert the target to queryparam for all 4 of these calls or just these two?

Reply:
I think it's fine to leave the target in the body for the other calls where we already have a body

Reply:
sure thing!